packages/graphql/src/contracts.js

```diff
@@ -413,7 +411,7 @@ function setMetaMask() {
     context.ognExec = context.ognMM
     context.tokens.forEach(token => (token.contractExec = token.contractMM))
     context.daiExchangeExec = context.daiExchangeMM
-  } else {
+  } else if (!shouldUseMobileBridge()) {
```

Comment:
nit:
 - might be cleaner check for shouldUseMobileBridge() at the top of setMetaMask method and return right away
 - add a comment explaining (if I understand correctly) that this is the case for being in an environment that is not origin mobile and not metamask.

Reply:
Yup good point, early exits are nicer, also adding the comment.